main.nf

```diff
@@ -464,134 +519,205 @@ dwi_for_test_gibbs
     .into{dwi_for_eddy; dwi_for_topup; dwi_for_eddy_topup; dwi_for_test_eddy_topup}
```

Comment:
Processes below don't have the tag ("_" or "_rev_") has input. Don't remove it here, it is still needed to decipher forward from reverse in channels operations. Once the comment for Prepare_for_Topup is addressed, I'll be able to point where and how we should remove the tag.